ehitchhiking-ui/src/app/shared/components/no-data/no-data.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="no-data-wrapper">
+	<div [ngClass]="defineSize(size)">
+		<i class="material-icons">
+			work_off
+		</i>
+		<p class="no-data-text">No data!</p>
```

Comment:
By default should display "No Data" message. It should be possible to change the message.

Reply:
done